### ISSUE TITLE
TypeError: Cannot convert undefined or null to object

### ISSUE THREAD

**Issue by samtgarson** [nordnet/sass-variable-loader#5]
Received the following error using this command: 
```js
import styleVars from 'sass-variable-loader!../assets/mixins/layout.sass'
```

with this SASS file:

```sass
$body-margin: 65px
$nav-height: 100px
$breathing-space: 40px
$mega-nav-height: 500px
```

```
ERROR in ./~/sass-variable-loader/dist!./app/assets/mixins/layout.sass
Module build failed: TypeError: Cannot convert undefined or null to object
    at parseVariables (/opt/node_modules/sass-variable-loader/dist/parse-variables.js:61:24)
    at Object.sassVariableLoader (/opt/node_modules/sass-variable-loader/dist/index.js:20:48)
 @ ./~/babel-loader!./~/vue-loader/lib/selector.js?type=script&index=0!./app/components/sub-nav.vue 11:14-74
```

Don't have time right now to take a look at it but will take a deeper look later.

**Comment by samtgarson** [comment]
I had to add `!!` before the import (missed that in the docs first time...)

But I also had to add a `;` to the end of the variable declarations. 

Do you think it would be easy to add support for `sass` format as well as `scss` format? (i.e. no semi colons...)

**Comment by LeBlaaanc** [comment]
I have a similar error if I pass parseVariables an empty scss file, I get that this is an edge case but it's also probably not the desired result?

**Comment by JohnPhoto** [comment]
I'm not very familiar with sass-format but I'm guessing this line https://github.com/nordnet/sass-variable-loader/blob/master/src/get-variables.js#L4 is the culprit, (notice the last `;` in the regexp).

Maybe it is enought to just make the `;` optional in the regexp to support your usecase @samtgarson ?

**Comment by JohnPhoto** [comment]
Should be fixed in `v0.1.0`
